app/models/manageiq/providers/google/cloud_manager/cloud_database.rb

```diff
@@ -1,2 +1,49 @@
 class ManageIQ::Providers::Google::CloudManager::CloudDatabase < ::CloudDatabase
+  supports :create
+  supports :delete
+
+  def self.params_for_create
+    {
+      :fields => [
+        {
+          :component => 'text-field',
+          :id        => 'name',
+          :name      => 'name',
+          :label     => _('Cloud Database Name'),
+        },
+        {
+          :component    => 'select',
+          :name         => 'tier',
+          :id           => 'tier',
+          :label        => _('Tier'),
+          :includeEmpty => true,
+          :isRequired   => true,
+          :options      => ManageIQ::Providers::Google::CloudManager::CloudDatabaseFlavor.all.map do |db|
```

Comment:
CloudDatabaseFlavors are collected during refresh so if you have more than one provider the list will be duplicated.

Recommend passing in the ems like we do with cloud_volumes: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/app/models/manageiq/providers/openstack/storage_manager/cinder_manager/cloud_volume.rb#L17
Then you can do `ems.cloud_database_flavors.map ...`